### ISSUE TITLE
Planning to add complete set of solution of Problem Solving of HackerRank

### ISSUE THREAD

**Issue by TechnoBlogger14o3** [TechnoBlogger14o3/HackerRank-Problem-Solving-Solutions#1]
Need contribution in completing Problem Solving of HackerRank. 

[Here is the link](https://www.hackerrank.com/domains/data-structures?badge_type=problem-solving)

**Comment by shashwat9kumar** [comment]
Hey mate, do you encourage PRs on your repository?

**Comment by TechnoBlogger14o3** [comment]
> Hey mate, do you encourage PRs on your repository?

Absolutely, but lets keep it organized. Make a new branch, and raise PR. 
Appreciate It.

**Comment by maanpatel2492** [comment]
Hey, We have to give a solution to Problem Solving of HackerRank right?
Can I contribute it in c++?

**Comment by TechnoBlogger14o3** [comment]
@maanpatel2492 
Sure, it is open for all languages.

**Comment by sapoorv1227** [comment]
Will this contribution be added to Hacktober Fest or not because its telling that this repository in not participating in Hacktober Fest??

**Comment by TechnoBlogger14o3** [comment]
@sapoorv1227 Yes it, I'm labelling PR as hacktoberfest-accepted

**Comment by sapoorv1227** [comment]
> @sapoorv1227 Yes it, I'm labelling PR as hacktoberfest-accepted

Okay, Thanks. I have already raised a PR in a new branch. Please do have a look.

**Comment by TechnoBlogger14o3** [comment]
> > @sapoorv1227 Yes it, I'm labelling PR as hacktoberfest-accepted
> 
> Okay, Thanks. I have already raised a PR in a new branch. Please do have a look.

@sapoorv1227 Hey, I don't see any PRs, please raise it again. Thanks.

**Comment by NavtikaKumar** [comment]
pull request created just now on a new branch nav. kindly review and approve

**Comment by NavtikaKumar** [comment]
another pull request has been created for "Medium" folder. please review and approve!
